clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -71,11 +66,13 @@
   @Getter private final List<String> omitNamespaces;
   private final List<KubernetesKind> kinds;
   private final Map<KubernetesKind, InvalidKindReason> omitKinds;
+  private final Map<KubernetesKind, InvalidKindReason> omitKindsComputed = new HashMap<>();
   @Getter private final List<CustomKubernetesResource> customResources;
 
   @Getter private final String kubectlExecutable;
   @Getter private final Integer kubectlRequestTimeoutSeconds;
   @Getter private final String kubeconfigFile;
+  @Getter private final String kubeconfigFileHash;
```

Comment:
Do we need the `@Getter` here?  I'd think this should be an implementation detail of the class.